index.md

```diff
@@ -99,3 +105,28 @@ Kiali is open source and released under [Apache License v2](https://www.apache.o
 
 [Kiali on Irc](irc://irc.freenode.net/#kiali)
 
+
+## Appendix
+
+<h3 id="install-istio">Installing Istio</h3>
+
+
+You can use the following script to install Istio. The user installing it needs cluster-admin rights (e.g. the admin user from above).
+
```

Comment:
nit : "from above" part might be a bit out of context here, because this is much further in the doc from the part where it is mentioned